### PR TITLE
XML examples with named ID's for FGM / CPIS

### PR DESCRIPTION
**Pull request by APageNHS**
FGM examples were stored only in simplifier, in xml, and with uuid's for the id element
CPIS  examples were stored only in simplifier, in json, and with uuid's for the id element

This PR adds them to a folder per resource type, with a file and id naming convention of [ProgrammeAcronym]-[WhatsItsAnExampleOf]-Example.xml - all examples will be in xml as per IOPS standard guidelines

### CODE REVIEW

**Comment by APageNHS** [comment]
NOTE - I've asked RM to configure the IOPs quality control checks on this repo - I will hold off mergine until this is configured and confirmed working.

File: Bundle/FGM-BundleWithFoundFlag-Example.xml
Comment (vickyjaiswal0308) [inline Bundle/FGM-BundleWithFoundFlag-Example.xml:1]: There's a Spelling mistake in id value , there's a 'l' between found word

Comment (APageNHS) [reply Bundle/FGM-BundleWithFoundFlag-Example.xml:1]: I'd like to say "I did that on purpose, to see if you were paying attention"... Good spot vicky!

File: OperationOutcome/CPIS-AccessDenied-Example.xml
Comment (vickyjaiswal0308) [inline OperationOutcome/CPIS-AccessDenied-Example.xml:1]: Don't we need a named id here?

Comment (APageNHS) [reply OperationOutcome/CPIS-AccessDenied-Example.xml:1]: Yup, good spot. I'll leave this one to allow us to check the quality control works, then fix

**Review by ryma2fhir — APPROVED** [review]

**Pull request merged**